pkg/platform/api/buildplanner/model/buildplan.go

```diff
@@ -333,6 +333,19 @@ func ProcessProjectError(project *Project, fallbackMessage string) error {
 	return errs.New(fallbackMessage)
 }
 
+type ProjectCreatedError struct{ Type string }
+
+func (p *ProjectCreatedError) Error() string { return "ProjectCreatedError" }
+
+func ProcessProjectCreatedError(pcErr *projectCreated, fallbackMessage string) error {
+	switch pcErr.Type {
+	case AlreadyExistsErrorType, ForbiddenErrorType:
+		return &ProjectCreatedError{pcErr.Type}
+	default:
+		return errs.New(fallbackMessage)
+	}
+}
```

Comment:
For most of the error types returned from the buildplanner we want to ensure that the `message` field is being forwarded to the user. This is true for all of them, like a `forbidden` error but for parsing and validation errors we should forward the message. I don't think this will happen with the current implementation.